src/components/ImportRecordsFlow/stepFour/statsCount.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import StyledStatsCardDescription from './styled/statsCardDescription';
+import XCircle from '../icons/xCircle';
+import CheckCircle from '../icons/checkCircle';
+import RenderIf from '../../RenderIf';
+
+export default function StatsCount(props) {
+    const { type, amount } = props;
+    return (
+        <>
+            <RenderIf isTrue={type === 'success'}>
+                <CheckCircle />
+            </RenderIf>
+            <RenderIf isTrue={type === 'error'}>
+                <XCircle />
+            </RenderIf>
+            <StyledStatsCardDescription>
+                {amount} {type === 'success' ? 'Successful' : 'Errors'}
```

Comment:
here we need to do plurarization or avoiding it like the first solution but first talk to @TahimiLeonBravo about this
https://2ality.com/2019/12/intl-pluralrules.html